pydoctor/templatewriter/pages/__init__.py

```diff
@@ -30,34 +31,125 @@ def signature(function: model.Function) -> str:
     return str(function.signature)
 
 class DocGetter:
-    def get(self, ob, summary=False):
+    """L{epydoc2stan} bridge."""
+    def get(self, ob:model.Documentable, summary:bool=False) -> Tag:
         if summary:
             return epydoc2stan.format_summary(ob)
         else:
-            doc = epydoc2stan.format_docstring(ob)
-            typ = epydoc2stan.type2stan(ob)
-            if typ is None:
-                return doc
+            return epydoc2stan.format_docstring(ob)
+    def get_type(self, ob: model.Documentable) -> Optional[Tag]:
+        return epydoc2stan.type2stan(ob)
+
+class BaseElement(Element, abc.ABC):
```

Comment:
In general, `BaseSomething` is a superclass of `Something` instead of the other way around. See for example `BaseException`.

What makes this different from other elements? I guess it has something to do with templates, so maybe include that in the name instead.